lib/util.js

```diff
@@ -64,5 +64,13 @@ module.exports = {
     }
 
     return this.arrayToCSV(options, ['fields', 'filterValues']);
-  }
+  },
+
+  nextPageType: function(url) {
+    var type = 'token';
+    if(requestedUrl.indexOf('asset') < 0) {
```

Comment:
isn't this backward? `requestedUrl.indexOf('asset') !== -1`

Reply:
good catch.. fixed it